### ISSUE TITLE
Why my result is such unclear? any body have the experiments?

### ISSUE THREAD

**Issue by RYHSmmc** [showlab/MotionDirector#7]

**Comment by XiaominLi1997** [comment]
I also meet the same problem.
1. given:
Prompt: A person is riding a bicycle past the Eiffel Tower.

**Comment by ruizhaocv** [comment]
Hi Xiaomin. Thanks for the feedback. How about other checkpoints? Like [https://github.com/showlab/MotionDirector#motiondirector-trained-on-a-single-video]. Generally, setting the same seed as listed in the readme will generate the same result as shown.

**Comment by XiaominLi1997** [comment]
> Hi Xiaomin. Thanks for the feedback. How about other checkpoints? Like (https://github.com/showlab/MotionDirector#motiondirector-trained-on-a-single-video). Generally, setting the same seed as listed in the readme will generate the same result as shown.

Yep, results from training on a single video are the same. thx

**Comment by XiaominLi1997** [comment]
> Nice. Maybe I confused the checkpoints of the riding bicycle. Will check that.

hi, I found a new problem, a train on a single video (prompt: A person is skateboarding)

Given the same seed=6668889 and prompt="A panda is skateboarding." during both training and inference.
1. sample a video **during training** with ckpt-300
the result is pretty good.
https://github.com/showlab/MotionDirector/assets/25433111/a35240a6-4b65-41d8-906c-b15f1f300741

**however,**
2. sample a video **during inference** with ckpt-300
the performance is bad.

https://github.com/showlab/MotionDirector/assets/25433111/ec55528f-6a94-49af-84e6-f79134d4dc58

Could you please check the inference code or check the reason (maybe the hyper-parameters)? I and my co-worker met the same problem.

Inference hyper-parameters I used:
`"args": [
                "--model",
                "/15764332239/pretrained_models/text-to-video-ms-1.7b",
                "--prompt",
                "A panda is skateboarding.",
                "--checkpoint_folder",
                "./outputs/train/skateboard-single-video",
                "--checkpoint_index",
                "300",
                "--noise_prior",
                "0.5",
                "--seed",
                "6668889"
            ],`

**Comment by ruizhaocv** [comment]
How about inference with checkpoint_index=150 look like?

**Comment by ruizhaocv** [comment]
For faster convergence, we set a large learning rate, which may cause instability in the late training steps. If you want a more stable but slower training, you can try to reduce the learning rate. Enjoy exploring the optimal hyperparameters for your own training task.

**Comment by ruizhaocv** [comment]
Setting the seed fixed for inference will make sure to generate the same results. However, setting the same random seed does not mean you will get the exactly same results in the inference stage and training stage.  Because every time the seed is called in the training stage, it will change.

**Comment by XiaominLi1997** [comment]
> Setting the seed fixed for inference will make sure to generate the same results. However, setting the same random seed does not mean you will get the exactly same results in the inference stage and training stage. Because every time the seed is called in the training stage, it will change.

Thanks, I just mistakenly thought the seed below was the validation seed.

![image](https://github.com/showlab/MotionDirector/assets/25433111/9dd590fb-ee8b-447d-972f-908f30da1022)

Thanks again for your nice reply.

**Comment by ruizhaocv** [comment]
Thanks for pointing this out. I have deleted this confusing item.
